cpplint/cpplint_unittest.py

```diff
@@ -37,6 +37,7 @@
 import os
 import random
 import re
+import six
```

Comment:
this is a third party import, so it should be after unittest but before cpplint

Reply:
Removed.